targets/XPlatCppSdk/source/cppsdk/source/playfab/PlayFabCallRequestContainerBase.cpp

```diff
@@ -0,0 +1,68 @@
+#include <stdafx.h>
+
+#include <playfab/PlayFabCallRequestContainer.h>
+
+namespace PlayFab
+{
+    CallRequestContainerBase::CallRequestContainerBase(
+        std::string url,
+        const std::unordered_map<std::string, std::string>& headers,
```

Comment:
> const std::unordered_map<std::string, std::string>& headers, [](start = 7, length = 61)

So we might be in trouble if someone cleans up this structure before we're done using it.

Reply:
We create and initialize own own copy of it as a class member

---
In reply to: [220343802](https://github.com/PlayFab/SDKGenerator/pull/295#discussion_r220343802) [](ancestors = 220343802)